### ISSUE TITLE
Tons of errors when opening in Visual studio 2019

### ISSUE THREAD

**Issue by ghost** [ErLinErYi/PlantsVsZombies#14]
i get hundreds of errors when i open in visual studio 2019 but only in the Plants vs zombies project not anything else

**Comment by ErLinErYi** [comment]
you may need game resource



---Original---
From: ***@***.***&gt;
Date: Thu, Aug 19, 2021 02:10 AM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [ErLinErYi/PlantsVsZombies] Tons of errors when opening in Visual studio 2019 (#14)




 
i get hundreds of errors when i open in visual studio 2019 but only in the Plants vs zombies project not anything else
 
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
 
  	
 	 		  			从QQ邮箱发来的超大附件 		 	
 	 		  	 	 		 			 		
 		 			 资源文件_0619.rar (47.01M, 无限期) 			 			 				进入下载页面：https://mail.qq.com/cgi-bin/ftnExs_download?k=9a9d5e61db4b5b4ba8ee1661666663351341c86364666335404d5e5856565307510415515003521851510104495e0002064f5d020050020d5d500b51000776358dd7bc87def685a3e28683d73b5655045c4c4a0016725acd42574b27423574caa5b58eaa7a1244226953&t=exs_ftn_download&code=eb8adfc5

**Comment by ghost** [comment]
Ill try but it was saying in visual studio that you are not allowed to use these character on many differant cpp and h files

**Comment by ErLinErYi** [comment]
I see. Because some variables in the code use Chinese characters and your computer language is English, an error has occurred. You may need to change the Chinese characters in the code to English.



---Original---
From: ***@***.***&gt;
Date: Thu, Aug 19, 2021 19:01 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [ErLinErYi/PlantsVsZombies] Tons of errors when opening in Visual studio 2019 (#14)




 
Ill try but it was saying in visual studio that you are not allowed to use these character on many differant cpp and h files
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by ghost** [comment]
Okay i will try also amazing project im trying to get better at c++ with this project

**Comment by ghost** [comment]
It gave error for this line in the code 
        ÏÔÊ¾ÐÅÏ¢ = 1,
	È«ÆÁ,
	¸ßÖ¡ÂÊ,
	Êó±êÒþ²Ø,
	À­ÉìÏÔÊ¾,
	»ºÈë¶¯»­,
	´¹Ö±Í¬²½

**Comment by ghost** [comment]
not sure why its like that

**Comment by ghost** [comment]
some of the comments in the code are like that too

**Comment by ghost** [comment]
maybe glitched code

**Comment by ghost** [comment]
would be nice if you can send english translated version or something

**Comment by ErLinErYi** [comment]
Because Chinese characters cannot be parsed normally, they are displayed as garbled codes. Those garbled codes are enumeration types. In fact, you can replace them with English at will. If you can't finish it yourself, wait for me to update it.



---Original---
From: ***@***.***&gt;
Date: Thu, Aug 19, 2021 23:02 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [ErLinErYi/PlantsVsZombies] Tons of errors when opening in Visual studio 2019 (#14)




 
It gave error for this line in the code
 ÏÔÊ¾ÐÅÏ¢ = 1,
 È«ÆÁ,
 ¸ßÖ¡ÂÊ,
 Êó±êÒþ²Ø,
 À­ÉìÏÔÊ¾,
 »ºÈë¶¯»­,
 ´¹Ö±Í¬²½
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by ghost** [comment]
yes but in your code is it chinese because if it is i can translate it to english

**Comment by ghost** [comment]
actually in github the code is english so i just type the english version in the code but one line called Display information = 1 is giving an error on the word information

**Comment by ghost** [comment]
It also has issues importing other files such as .h
